tasks/jenkins.rake

```diff
@@ -3,17 +3,24 @@ require 'English'
 begin
   require "ci/reporter/rake/test_unit"
   namespace :jenkins do
+    desc 'Sets up pre-CI environment for testing'
     task :unit => ["jenkins:setup:test_unit", 'rake:test']
 
     namespace :setup do
+      desc 'Sets up pre-CI environment for Minitest'
       task :pre_ci do
         ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
         gem 'ci_reporter'
       end
-      task :minitest  => [:pre_ci, "ci:setup:minitest"]
+
+      desc 'Sets up pre-CI environment for MiniTest'
+      task :minitest => [:pre_ci, "ci:setup:minitest"]
+
+      desc 'Sets up pre-CI environment for Test::Unit'
       task :test_unit => [:pre_ci, "ci:setup:testunit"]
     end
 
+    desc 'Run RuboCop analysis'
```

Comment:
We no longer use this task so it's better to drop it. This is how we really invoke it now:

https://github.com/theforeman/jenkins-jobs/blob/a44d192c1549be46b3f0d85c254f4247c4ec9331/theforeman.org/pipelines/test/smart-proxy.groovy#L47

Note that if you drop this task, you no longer need the `require 'English'` at the top.